pint_server/tests/loadtest/loadtest.py

```diff
@@ -0,0 +1,85 @@
+from concurrent.futures import ThreadPoolExecutor
+from datetime import datetime, timedelta
+import argparse
+import random
+import requests
+import sys
+import time
+
+'''
+To run this program, 
+python loadtest.py --help displays the options the program takes
+If you do not provide any options, there are default values for the options
+python loadtest.py --num_requests 100 --duration 60 --url http://localhost:5000   
+'''
+
+urls_to_hit = ["/v1/providers",
+               "/v1/amazon/images", "/v1/amazon/servers",
+               "/v1/google/images", "/v1/google/servers",
+               "/v1/oracle/images", "/v1/oracle/servers",
+               "/v1/microsoft/images", "/v1/microsoft/servers"
+               "/v1/amazon/regions",
+               "/v1/amazon/us-east-2/images",
+               "/v1/microsoft/regions",
+               "/v1/providers.xml",
+               "/v1/alibaba/images.xml"]
+
+def task(base_url):
+    print("Executing the Task")
+    url = base_url + random.choice(urls_to_hit)
+    requests.get(url, verify=False)
+    #print("Task Executed {}".format(threading.current_thread()))
+
+def main(argv):
+    #handle command line arguments
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-n', '--num_requests',
+                        required=False,
+                        type=int,
+                        default=100,
+                        dest="num_requests",
+                        help="Number of requests to hit the API")
+    parser.add_argument('-d', '--duration',
+                        required=False,
+                        type=int,
+                        default=60,
+                        dest="duration",
+                        help="Duration in seconds")
```

Comment:
I think minutes would be better as load tests usually run in minutes.

Reply:
Updated the script to use minutes for duration